spark/src/main/scala/io/substrait/spark/ToSubstraitType.scala

```diff
@@ -29,6 +29,8 @@ import scala.collection.JavaConverters.asScalaBufferConverter
 private class ToSparkType
   extends TypeVisitor.TypeThrowsVisitor[DataType, RuntimeException]("Unknown expression type.") {
 
+  override def visit(expr: Type.I8): DataType = ByteType
+  override def visit(expr: Type.I16): DataType = ShortType
```

Comment:
while you're at it, mind adding these also to ToSparkType?

Reply:
This is in `ToSparkType`.  The conversion in `ToSubstraitType` is already there further down this file. It converts both ways.

Reply:
Ah right, all good then!